packages/example/src/components/PanEvent/panEventHandler.ts

```diff
@@ -0,0 +1,86 @@
+import { HWEvent } from 'react-native';
+import RemoteControlManager from '../remote-control/RemoteControlManager';
+import { SupportedKeys } from '../remote-control/SupportedKeys';
+import { PanEventInterface } from './PanEvent.interface';
+import { throttle } from '../../utils/throttle';
+import { repeat } from '../../utils/repeat';
+
+const GRID_SIZE = 1920;
+const NUMBER_OF_COLUMNS = 5;
+
+const PanEvent: PanEventInterface = {
+  orientation: undefined,
+  lastIndex: 0,
+  reset: () => {
+    PanEvent.orientation = undefined;
+    PanEvent.lastIndex = 0;
+  },
+  handlePanEvent: ({ x, y }: { x: number; y: number }) => {
+    const newIndex = getGridCoordinates(x, y);
+    if (!newIndex) return;
+    moveFocus(newIndex);
+  },
+};
+
+export const panEventHandler = (event: HWEvent) => {
+  throttle(() => {
+    if (event.eventType === 'pan') {
+      if (!event.body) return;
+      if (event.body.state === 'Began') {
+        PanEvent.reset();
+      }
+      if (event.body.state === 'Changed') {
+        PanEvent.handlePanEvent({ x: event.body.x, y: event.body.y });
+      }
+    }
+  }, 30)();
```

Comment:
you could take this one as a constant (a different one I guess? It doesn't represent the same thing as the other `30`, right?)